### PR TITLE
Updating Helm to 1.9.4

### PR DESCRIPTION
**Pull request by joekrejci**
Just kidding its actually 0.9.1 :)

### CODE REVIEW

File: charts/qdrant/Chart.yaml [hunk 1]
```diff
@@ -13,30 +13,11 @@ maintainers:
     url: https://github.com/qdrant
 icon: https://qdrant.github.io/qdrant-helm/logo_with_text.svg
 type: application
-version: 0.9.0
-appVersion: v1.9.1
+version: 0.9.4
```
Comment (mac-chaffee) [inline charts/qdrant/Chart.yaml:16]: The version is entirely separate from the app version, so it should just be bumped according to normal semver rules, to 0.9.1

Comment (joekrejci) [reply charts/qdrant/Chart.yaml]: Pushed version fix

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [qdrant-0.9.1](https://github.com/qdrant/qdrant-helm/tree/qdrant-0.9.4) (2024-06-10)
```
Comment (mac-chaffee) [inline CHANGELOG.md:3]: another one here

File: charts/qdrant/CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,8 @@
 # Changelog
 
-## [qdrant-0.9.0](https://github.com/qdrant/qdrant-helm/tree/qdrant-0.9.0) (2024-05-06)
+## [qdrant-0.9.1](https://github.com/qdrant/qdrant-helm/tree/qdrant-0.9.4) (2024-06-10)
```
[the remark this answers is not in the document]
Comment (joekrejci) [reply charts/qdrant/CHANGELOG.md]: ☕

**Review by mac-chaffee — APPROVED** [review]

**Pull request merged**